File: easterEggs/hotDogs/Order.java
import java.util.Scanner;

public class Order{

        public static void main (String [] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter the First intger:");
    int num1= sc.nextInt();
    
System.out.println("Enter the Second intger:");
    int num2= sc.nextInt();
 System.out.println("Enter the Third intger:");
    int num3= sc.nextInt();



    if(num1>=num3 && num2>=num3){

              if (num1>=num2){

             System.out.printf("The order is %d %d % d",num3,num2,num1);
        } else {
                System.out.printf("The order is %d %d % d",num3,num1,num2);
      }


    }else if (num1>=num2 && num3>=num2){

             if (num1>= num3){

            System.out.printf("The order is %d %d % d",num2,num3,num1);
        } else {

                      System.out.printf("The order is %d %d % d",num2,num1,num3);
        }



    }else if (num2>=num3 && num1>=num3){

            if (num2>= num1){

            System.out.printf("The order is %d %d % d",num3,num1,num2);
        } else {

            System.out.printf("The order is %d %d % d",num3,num2,num1);

}



    }else if (num2>=num1 && num3>=num1){

            if (num2>= num3){

            System.out.printf("The order is %d %d % d",num1,num3,num2);
        } else {

            System.out.printf("The order is %d %d % d",num1,num2,num3);
    }



    }  else if (num3>=num1 && num2>=num1){

            if (num3> num2){

            System.out.printf("The order is %d %d % d",num1,num2,num3);
        }else {

        System.out.printf("The order is %d %d % d",num1,num3,num2);
    }



    } else if (num3>num2 && num1>num2){

            if (num3>= num1){

            System.out.printf("The order is %d %d % d",num2,num1,num3);
        } else {

     System.out.printf("The order is %d %d % d",num2,num2,num1);
}



    }else{

System.out.print("that is strange");}
        











      
    






    }






}

File: Python eastereggs/hotdoggs/order.py
num1 = int(input("Enter the First integer: "))
num2 = int(input("Enter the second integer: "))
num3 = int(input("Enter the third integer: "))

if num1 >= num3 and num2 >= num3:
    if num1 >= num2:
        print(f"The order is {num3} {num2} {num1}")
    else:
        print(f"The order is {num3} {num1} {num2}")

elif num1 >= num2 and num3 >= num2:
    if num1 >= num3:
        print(f"The order is {num2} {num3} {num1}")
    else:
        print(f"The order is {num2} {num1} {num3}")

elif num2 >= num3 and num1 >= num3:
    if num2 >= num1:
        print(f"The order is {num3} {num1} {num2}")
    else:
        print(f"The order is {num3} {num2} {num1}")

elif num2 >= num1 and num3 >= num1:
    if num2 >= num3:
        print(f"The order is {num1} {num3} {num2}")
    else:
        print(f"The order is {num1} {num2} {num3}")

elif num3 >= num1 and num2 >= num1:
    if num3 > num2:
        print(f"The order is {num1} {num2} {num3}")
    else:
        print(f"The order is {num1} {num3} {num2}")

elif num3 > num2 and num1 > num2:
    if num3 >= num1:
        print(f"The order is {num2} {num1} {num3}")
    else:
        
        print(f"The order is {num2} {num3} {num1}")

else:
    print("that is strange")
